### PR TITLE
Improve user's knowledge on failures (?)

### PR DESCRIPTION
**Pull request by mirca**
Not sure if that is desirable, but I feel the need for being more verbose about failures. Probably that's the PyKE-effect on me ;)

### CODE REVIEW

File: k2qc/core.py [hunk 1]
```diff
@@ -120,7 +127,12 @@ def verify_positive_flux(self):
             warnings.simplefilter("ignore")  # ignore NaN comparison warnings
             mask = self.tpf[1].data['QUALITY'] == 0
             flux_plus_bkg = self.tpf[1].data['FLUX'][mask] + self.tpf[1].data['FLUX_BKG'][mask]
-            assert (flux_plus_bkg < 0).sum() == 0
+            if (flux_plus_bkg < 0).sum() > 0:
+                badpixels = np.argwhere(flux_plus_bkg < 0)
+                for pix in badpixels:
+                    print("Negative flux values found at frame {},"
```
Comment (barentsen) [inline k2qc/core.py]: Can we make it say "(FLUX + FLUX_BKG) < 0" instead of "Negative flux values", just to be more explicit?

Comment (mirca) [reply k2qc/core.py]: Ok, done! I also fixed the issue regarding the frame number (before it was printing the frame number relative to ``quality==0`` frames; now it prints the actual frame number).

**Comment by barentsen** [comment]
This looks great.  I'm happy for this to be merged now, please go ahead @mirca!  You get a :strawberry:.

**Comment by mirca** [comment]
🍓 🍓 🍓 🍓 🍓 🍓

**Pull request merged**